lib/strategy.js

```diff
@@ -117,29 +117,12 @@ util.inherits(Strategy, OAuth2Strategy);
  */
 Strategy.prototype.userProfile = function(accessToken, done) {
   var self = this;
-  this._oauth2._useAuthorizationHeaderForGET = true;
-  this._oauth2.get(this._userProfileURL, accessToken, function (err, body, res) {
```

Comment:
You remove the opening bracket here which causes the following error...

```
/data/node_modules/passport-keycloak-oauth2-oidc/lib/strategy.js:142
  });
   ^

SyntaxError: Unexpected token ')'
```

Reply:
Also, this doesn't appear to be a bug. The docs are simply missing the 'scope' attribute...

https://github.com/louie007/passport-keycloak-oauth2-oidc/pull/8/files